### PR TITLE
7 spawns

### LINKED ISSUE #7: Spawns
TODO:

- [x]  Cameras facing center
- [ ] three buzzers per player

Buzzers need to have proper authority and constant spawn locations

### PR DESCRIPTION
**Pull request by Yetti95**

### CODE REVIEW

**Comment by Yetti95** [comment]
Just adjusted spawn locations and rotations

File: scenes/Level.tscn [hunk 1]
```diff
@@ -197,19 +194,19 @@ spawn_limit = 4
 [node name="PlayerSpawners" type="Node3D" parent="."]
 
 [node name="0" type="Node3D" parent="PlayerSpawners" groups=["PlayerSpawnPoint"]]
-transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0.00505245, 2.28164, 4.0542)
+transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, -0.182017, 2.33142, 3.83535)
 metadata/spawned = false
 
 [node name="1" type="Node3D" parent="PlayerSpawners" groups=["PlayerSpawnPoint"]]
-transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0.00505245, 2.28164, -4.91662)
+transform = Transform3D(-1, 0, -8.74228e-08, 0, 1, 0, 8.74228e-08, 0, -1, -0.0340034, 2.28164, -3.93498)
```
Comment (isaacanielson) [inline scenes/Level.tscn]: Can round to 0

File: scenes/Level.tscn [hunk 2]
```diff
@@ -197,19 +194,19 @@ spawn_limit = 4
...
 metadata/spawned = false
 
 [node name="2" type="Node3D" parent="PlayerSpawners" groups=["PlayerSpawnPoint"]]
-transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 5.24074, 2.18877, 0.00565672)
+transform = Transform3D(-4.37114e-08, 0, 1, 0, 1, 0, -1, 0, -4.37114e-08, 3.47767, 2.18806, 0.0655866)
```
Comment (isaacanielson) [inline scenes/Level.tscn]: Can round to 0

File: scenes/Level.tscn [hunk 3]
```diff
@@ -197,19 +194,19 @@ spawn_limit = 4
...
 metadata/spawned = false
 
 [node name="3" type="Node3D" parent="PlayerSpawners" groups=["PlayerSpawnPoint"]]
-transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, -5.12633, 1.95762, -0.0725052)
+transform = Transform3D(-4.37114e-08, 0, -1, 0, 1, 0, 1, 0, -4.37114e-08, -4.64265, 1.96027, -0.176082)
```
Comment (isaacanielson) [inline scenes/Level.tscn]: Can round to 0

File: scenes/Level.tscn [hunk 4]
```diff
@@ -197,19 +194,19 @@ spawn_limit = 4
 [node name="PlayerSpawners" type="Node3D" parent="."]
 
 [node name="0" type="Node3D" parent="PlayerSpawners" groups=["PlayerSpawnPoint"]]
-transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0.00505245, 2.28164, 4.0542)
+transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, -0.182017, 2.33142, 3.83535)
```
Comment (isaacanielson) [inline scenes/Level.tscn]: Can round to 0

**Pull request merged**